### ISSUE TITLE
The code of ArcFace is the same as that of SphereFace : (

### ISSUE THREAD

**Issue by WintrumWang** [4uiiurz1/keras-arcface#16]
And if the parameter m is set like 4, the label must be leaked, because the training accuracy, validation accracy and test accuracy are all 1.0 even in the 1st epoch!

**Comment by SerChirag** [comment]
**In case of ArcFace :**
target_logits = tf.cos(theta + self.m)

**In case of SphereFace :**
target_logits = tf.cos(self.m * theta)
